src/login/translation/languages/sv.json

```diff
@@ -366,15 +366,15 @@
   "resetpw.email-send-failure": "Misslyckades att skicka e-post, vänligen försök igen",
   "resetpw.email-throttled": "Återställ lösenordslänken har redan skickats, försök igen senare",
   "resetpw.enter-code": "skriv in koden",
-  "resetpw.expired-email-code": "E-postkoden har gått ut. Var god försök igen.",
-  "resetpw.expired-phone-code": "Telefonkoden har gått ut. Var god försök igen.",
+  "resetpw.expired-email-code": "Lösenord återställnings länken är inte längre giltig. Skicka länken igen.",
```

Comment:
Lösenord återställnings länken -> Lösenordåterställningslänken but maybe Länken för lösenordsåterställning is easier to parse :)